recipes/r-base/meta.yaml

```diff
@@ -63,7 +63,7 @@ requirements:
     - {{native}}bwidget        # [win]
     - {{native}}tktable        # [win]
     - {{native}}pango 1.40.*   # [not win]
-    - {{native}}cairo          # [win]
+    - cairo          # [win]
```

Comment:
You cannot link MSVC libraries with GCC libraries except in very exceptional circumstances. Even when it can be done incompatibilities render it very dangerous. The entire ecosystem of R must be built with GCC. That is why we have `m2w64-` packages.